### ISSUE TITLE
feature: accept an address, and ping it on demand

### ISSUE THREAD

**Issue by trombik** [trombik/esphome-component-ping#10]
when trouble-shooting weird network issues, it is useful to be able to `ping` an address from the device. I would like to `ping` an address, which is given on run-time, and see the results.

**Comment by nagyrobi** [comment]
yes that would be nice. allow the parameter `ip_address` to be templated. With the new [input_text](https://github.com/esphome/esphome/pull/3638) component we could reference to any inputted text via a lambda. Input validation would be required.
